### PR TITLE
Rerender for python 3.6 support

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@conda-forge/core if anyone can hit the green button, it would be fantastic. There is a chain of dependencies to rerender (arg <- cling <- twine...)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,3 +34,4 @@ extra:
   recipe-maintainers:
     - ericdill
     - janschulz
+    - SylvainCorlay
```
Comment (jankatins) [inline recipe/meta.yaml:37]: welcome :-)

**Comment by jankatins** [comment]
@SylvainCorlay do you want to become a member in twine/cling?

**Comment by SylvainCorlay** [comment]
Yes (assuming you meant clint!) I am already a member for cling :)

**Pull request merged**